tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
The hypev task learns on separate samples and the averaging is done only when evaluating. Load_hypev is therefore unusable in yesno. In my opinion the hypev task is outdated..